src/StructureFactors.jl

```diff
@@ -407,6 +423,7 @@ of `spin_traj`.
 function fft_spin_traj!(res::Array{ComplexF64}, spin_traj::Array{Vec3};
                         plan::Union{Nothing, FFTW.cFFTWPlan}=nothing)
     @assert size(res) == tuplejoin(3, size(spin_traj)) "fft_spins size not compatible with spin_traj size"
+    println("here")
```

Comment:
I think this line accidentally made it through

Reply:
Sorry. Thanks. Removed it.